### PR TITLE
Notify main actions via Slack#44

### PR DESCRIPTION
**Pull request by CongThuong**

### CODE REVIEW

File: api/controllers/preregistrationController.js [hunk 1]
```diff
@@ -80,6 +81,16 @@ exports.add = async (req, res) => {
       { referred: referrer.referred + 1 }
     );
   }
+  const sendSlack = async () => {
+    try {
+      const resp = await axios.post(`${webhook}`, {
+        text: `Someone registered successfully`,
```
Comment (jatitoam) [inline api/controllers/preregistrationController.js]: Please tell the id who registered

File: api/controllers/preregistrationController.js [hunk 2]
```diff
@@ -234,6 +245,16 @@ exports.activate = async (req, res) => {
       },
       { dateActivation: new Date() }
     );
+    const sendSlack = async () => {
+      try {
+        const resp = await axios.post(`${webhook}`, {
+          text: `Someone activated successfully`,
```
Comment (jatitoam) [inline api/controllers/preregistrationController.js]: Please tell the id who registered

File: api/controllers/preregistrationController.js [hunk 3]
```diff
@@ -302,6 +323,16 @@ exports.linkAesirX = async (req, res) => {
         dateAesirXAccount: new Date(),
       }
     );
+    const sendSlack = async () => {
+      try {
+        const resp = await axios.post(`${webhook}`, {
+          text: `Someone linked AesirX successfully`,
```
Comment (jatitoam) [inline api/controllers/preregistrationController.js]: Please tell the id and AesirX account who registered

**Review by jatitoam — CHANGES_REQUESTED** [review]

File: api/controllers/preregistrationController.js (hunk 3 above)
Comment (CongThuong) [reply api/controllers/preregistrationController.js]: Please take a look again

**Review by jatitoam — APPROVED** [review]

**Pull request merged**